kornia/geometry/calibration/distort.py

```diff
@@ -26,16 +28,16 @@ def tilt_projection(taux: torch.Tensor, tauy: torch.Tensor, return_inverse: bool
     sTx = torch.sin(taux)
```

Comment:
```suggestion
    sTx = taux.sin()
```
@Tchaikovic i would take this opportunity to review the existing code too.
This as an alternative, or expose sin, cos ? /cc @johnnv1

Reply:
@edgarriba @johnnv1 Let me know how you'd like me to proceed.

Reply:
It is valid to expose these in kornia.core too. Personally looks better read `torch.sin(xx)` then `xx.sin()`